packages/shared/components/inputs/PasswordInput.svelte

```diff
@@ -1,32 +1,32 @@
 <script lang="ts">
     import { Icon as IconEnum } from '@auxiliary/icon'
     import { localize } from '@core/i18n'
-    import { Icon, Text, TextInput } from 'shared/components'
+    import { Icon, InputType, Text, TextInput } from 'shared/components'
 
-    export let value = ''
-    export let classes = ''
-    export let strength = 0
+    export let value: string
+    export let classes: string | undefined = undefined
```

Comment:
I know this component is used in many other files, but I think it would be good to remove classes, since most of them are `mb-x` or `w-full`

Reply:
i think, it would be better to do it in a different PR :pray: